### DISCUSSION TITLE
V3 Menu with icons and badges (responsive)   example

### DISCUSSION THREAD

**Discussion by rhscjohn-dev** [saadeghi/daisyui#1934] [Q&A] [answered]
Version 3.x.x.  Playing around with example:  Menu with icons and badges (responsive)  and  the vertical response does not match the response in the docs.  Horizontal is correct.

When I use the code, the badges are aligned to the far right of the document.  Not to the immediate right of the of  icon as in the docs.

[Tailwind Playground](https://play.tailwindcss.com/COyepRRrjF)

Note:  changed `lg:menu-horizontal` to `md:menu-horizontal` in playground.

Not sure if this is a bug or a feature.

**Comment by saadeghi** [comment, marked as answer]
In reality your vertical menu width is not going to be that wide. You may want to add a `w-56 md:w-auto` or another width class to that.
The position is intentional and it looks better when the badge is aligned to end.  

If you want everything to align to left, you can use a class name like `auto-cols-max` to modify the grid:  
https://play.tailwindcss.com/D4LwWdThVF
